RecoLocalCalo/HcalRecAlgos/interface/EigenMatrixTypes.h

```diff
@@ -3,20 +3,22 @@
 
 #include <Eigen/Dense>
 
+using FLOAT = double;
```

Comment:
this is not really configurable.
What is the final target for this PR?
Is this just a technical step to get comparisons for an intermediate step from jenkins?
I thought that the goal was to switch to float for good.

Reply:
let's do one comparison with double
yes, ideally I want to move to the float.